superset/tags/api.py

```diff
@@ -384,3 +387,140 @@ def get_objects(self) -> Response:
                 exc_info=True,
             )
             return self.response_422(message=str(ex))
+
+    @expose("/favorite_status/", methods=("GET",))
+    @protect()
+    @safe
+    @statsd_metrics
+    @rison({"type": "array", "items": {"type": "integer"}})
+    @event_logger.log_this_with_context(
+        action=lambda self, *args, **kwargs: f"{self.__class__.__name__}"
+        f".favorite_status",
+        log_to_statsd=False,
+    )
+    def favorite_status(self, **kwargs: Any) -> Response:
+        """Favorite Stars for Dashboards
+        ---
+        get:
+          description: >-
+            Check favorited dashboards for current user
+          parameters:
+          - in: query
+            name: q
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/get_fav_star_ids_schema'
+          responses:
+            200:
+              description:
+              content:
+                application/json:
+                  schema:
+                    $ref: "#/components/schemas/GetFavStarIdsSchema"
+            400:
+              $ref: '#/components/responses/400'
+            401:
+              $ref: '#/components/responses/401'
+            404:
+              $ref: '#/components/responses/404'
+            500:
+              $ref: '#/components/responses/500'
+        """
+        requested_ids = kwargs["rison"]
+        tags = TagDAO.find_by_ids(requested_ids)
+        if not tags:
+            return self.response_404()
+
+        users_favorited_tags = TagDAO.favorited_ids(tags)
+        res = [
+            {"id": request_id, "value": request_id in users_favorited_tags}
```

Comment:
this will return response like 
```
[
  {
    "id": 1,
    "value": True,
  },
  {
    "id": 2,
    "value": False,
  },
]
```
Can we change `value` to be `favorite_status` just to be more readable

Reply:
Also for IDs that's not in the db yet, it will be included in the response, which I think is misleading.
Can we return 404 if there's any id in the requested_ids that does not exist

Reply:
This is integrated with the frontend hook, and this is the schema needed for it work properly with the ListView component